pkgs/top-level/all-packages.nix

```diff
@@ -16541,7 +16541,9 @@ in
 
   keyutils = callPackage ../os-specific/linux/keyutils { };
 
-  libselinux = callPackage ../os-specific/linux/libselinux { };
+  libselinux = callPackage ../os-specific/linux/libselinux {
+    python = python3;
+  };
```

Comment:
could we do this, but in the expression?
people could override `python3 = python`?

Reply:
The reason I did not do that is because it is overridden in `python-packages.nix` and it felt weird having `python3 = python` there.